### ISSUE TITLE
Theano Dependency

### ISSUE THREAD

**Issue by ahundt** [gaudetcj/DeepQuaternionNetworks#2]
I was looking through the code and ran it, is there any chance the theano dependency could be removed?  I was hoping to run this with the tf backend.

**Comment by gaudetcj** [comment]
Yes you just replace all the theano calls with tf calls. I have it done but
have not pushed it yet.
